server/server_media.go

```diff
@@ -211,12 +211,13 @@ func (s *MediaServer) MakeCommunityTokenImagesURL(communityID string, chainID ui
 	return u.String()
 }
 
-func (s *MediaServer) MakeCommunityImageURL(communityID, name string) string {
+func (s *MediaServer) MakeCommunityImageURL(communityID, name string, version uint32) string {
 	u := s.MakeBaseURL()
 	u.Path = communityDescriptionImagesPath
 	u.RawQuery = url.Values{
 		"communityID": {communityID},
 		"name":        {name},
+		"version":     {strconv.FormatUint(uint64(version), 10)},
```

Comment:
`uint64` here and `uint32` everywhere else?

Reply:
`FormatUint` accepts uint64 only :shrug:

Reply:
:roll_eyes: go doesn't have some sort of generics or method overloads? :open_mouth: